packages/itwinui-react/src/core/ComboBox/ComboBoxInput.tsx

```diff
@@ -225,6 +225,7 @@ export const ComboBoxInput = React.forwardRef((props, forwardedRef) => {
         autoCorrect='off'
         style={multiple ? { paddingLeft: tagContainerWidth + 18 } : {}}
         aria-describedby={multiple ? `${id}-selected-live` : undefined}
+        aria-label='Menu'
```

Comment:
How did you choose that the best label for this input is 'menu'?

Reply:
Generally speaking, labels should not be hardcoded for any component where we do not know how it will be used.

This combobox could be used for selecting fruits or cities or anything else, so the user is the one who should provide the label.

There is no change needed in the component; only the examples need to be updated. You can see existing examples that are already passing:
- https://github.com/iTwin/iTwinUI/blob/dev/examples/ComboBox.label.tsx#L46
- https://github.com/iTwin/iTwinUI/blob/dev/examples/List.combobox.tsx#L14

Reply:
Removed the change from the component and implemented a `<VisuallyHidden>` label in the `MultipleSelect` example. I followed a similar pattern to `List.combobox`.

Reply:
update other combobox examples? i'm sure this is not the only one failing